src/docs/framework-integration/react.md

```diff
@@ -75,13 +75,13 @@ If you do rename your React component library, be sure to change the `name` in t
 Now that the project structure is set up, we can install the React Output Target package in your Stencil component library. This package contains the React wrapper function that we will use to generate our React wrapped components inside a 'React component library'. To install the React Output Target package, run the following command in your **Stencil project directory**
 
 ```bash
-npm install @stencil/react-output-target
+npm install -D @stencil/react-output-target
 ```
 
 Or
 
 ```bash
-yarn add @stencil/react-output-target
+yarn add -D @stencil/react-output-target
```

Comment:
Could we use the long-hand version of `-D` please? That way, should someone who isn't familiar with the CLI has a better chance of understanding what we're asking them to do.
```suggestion
yarn add --dev @stencil/react-output-target
```